src/components/Navbar.js

```diff
@@ -1,14 +1,21 @@
 import React from 'react';
 import { NavLink } from 'react-router-dom';
+import { FaUser } from 'react-icons/fa';
+import styles from '../styles/Navbar.module.css';
 
 const Navbar = () => (
 
   <header>
-    <h1>BOOK STORE</h1>
-    <nav>
-      <NavLink to="/">Books</NavLink>
-      <NavLink to="/categories">Categories</NavLink>
-    </nav>
+    <div className={styles.header}>
+      <h1>BookStore CMS </h1>
+      <nav className={styles.NavLinkContainer}>
+        <NavLink className={styles.NavLink} to="/">Books</NavLink>
+        <NavLink className={styles.NavLink} to="/categories">Categories</NavLink>
+      </nav>
```

Comment:
Kindly use uppercase for the navlinks like on the zeplin design. 
<img width="312" alt="Screenshot 2023-03-18 at 2 09 47 PM" src="https://user-images.githubusercontent.com/25887253/226101766-accc7384-df23-45ef-8f93-76a49b9eb961.png">
<img width="643" alt="Screenshot 2023-03-18 at 2 11 23 PM" src="https://user-images.githubusercontent.com/25887253/226101779-5b0f7afa-564f-43ab-85a5-b4ba0409e9be.png">
<img width="752" alt="Screenshot 2023-03-18 at 2 13 45 PM" src="https://user-images.githubusercontent.com/25887253/226101872-814aba8f-5406-4620-bcae-8e584babe5d0.png">

Reply:
Thanks it's fixed